src/appengine/handlers/testcase_detail/show.py

```diff
@@ -19,6 +19,7 @@
 
 from flask import request
 import jinja2
+from markupsafe import escape
```

Comment:
We don't import functions, only packages/modules. See python styleguide for more details on this.